framework/auth/__init__.py

```diff
@@ -60,6 +63,13 @@ def register_unconfirmed(username, password, fullname):
             password=password,
             fullname=fullname)
         user.save()
+        mails.queue_mail(
+            to_addr=user.username,
+            mail=mails.NO_ADDON,
+            send_at=datetime.utcnow() + settings.NO_ADDON_WAIT_TIME,
+            user=user,
+            fullname=user.fullname
+        )
```

Comment:
Blinker signals are the preferred way to trigger side effects from model methods. Using them here would better decouple the mail-queueing from this User record logic.